app/views/users/_followees.html.haml

```diff
@@ -1,9 +1,7 @@
-.col-md-3
+.col-md-3.col-sm-3
   .media
     .media-left
-      = link_to(                                                                     |
-        image_tag(friend.icon.url(:thumb), :class => 'media-object img-circle', :width => 40), |
-        person_by_login_path(friend.login))                                          |
+      = link_to(image_tag(friend.icon.url(:thumb), :class => 'media-object img-circle', :style => "max-width:50px;max-height:50px;", :width => 50), person_by_login_path(friend.login))                                          
```

Comment:
another CSS thing. Ideally we don't do any inline styles and leave this for CSS to deal with